resnet/README.md

```diff
@@ -93,6 +93,9 @@ bazel-bin/resnet/resnet_main --train_data_path=cifar10/data_batch* \
                              --dataset='cifar10' \
                              --num_gpus=1
 
+# Note that training takes about 5 hours on a K40 GPU, but the training script will not produce any output. In the meantime you can check on progress using tensorboard:
```

Comment:
The length of time depends on the size of the model. Let's keep it simple:
"use tensorboard to see results".